examples/alpine-cli/build.sh

```diff
@@ -25,4 +24,4 @@ build_script | _docker run -i --name build-$BUILD_NUMBER alpine-example-build sh
 
 # since we build inside a container we need to explicitly copy the artifacts out
 # after building
-_docker cp build-$BUILD_NUMBER:/home/circleci/packages/build/x86_64/ .
+_docker cp -a build-$BUILD_NUMBER:/home/circleci/packages/build/. .
```

Comment:
Should this be recursive? `-R`

Reply:
Happy to be mistaken but I don't think there is a specific recursive flag for `docker cp`? According to [this](https://docs.docker.com/engine/reference/commandline/container_cp/) the `-a` flag performs the copy recursively.